apps/condo/domains/property/schema/CheckPropertyWithAddressExistService.js

```diff
@@ -5,13 +5,15 @@
 const { GQLCustomSchema } = require('@core/keystone/schema')
 const access = require('@condo/domains/property/access/CheckPropertyWithAddressExistService')
 const { Property } = require('@condo/domains/property/utils/serverSchema')
+const get = require('lodash/get')
+const { jsonAddressMetaValidator } = require('../utils/validation.utils')
 
 
 const CheckPropertyWithAddressExistService = new GQLCustomSchema('CheckPropertyWithAddressExistService', {
     types: [
         {
             access: true,
-            type: 'input CheckPropertyWithAddressExistInput { address: String! }',
+            type: 'input CheckPropertyWithAddressExistInput { addressMeta: JSON! }',
```

Comment:
add `address` and `addressMeta` in the future we will drop the addressMeta

Reply:
added a field (we are still using only addressMeta btw)